internals/cli/cmd_autostart.go

```diff
@@ -51,7 +54,7 @@ func (cmd cmdAutoStart) Execute(args []string) error {
 		return err
 	}
 
-	if _, err := cmd.wait(changeID); err != nil {
+	if _, err := cmd.wait(cmd.client, changeID); err != nil {
```

Comment:
If we're handing the client in, is there a benefit in making this a mixin?

Reply:
It's functionality shared by `autostart`, `replan`, `restart`, `start` and `stop` commands. I think a refactor here is possible (and desirable) but I'd rather leave it out of the scope of this PR.